resources/measures/ResidentialConstructionsWallsCMU/measure.rb

```diff
@@ -180,8 +184,16 @@ def run(model, runner, user_arguments)
       return false
     end
 
+    # Assume uninsulated wall properties if no exterior finish
+    if mat_ext_finish.name.include? "None"
+      unins_ext_finish = WallConstructions.get_exterior_finish_material("Vinyl, Light")
+      osb_thick_in = 0.5
```

Comment:
Why? Please add comment.

Reply:
There is a comment above: # Assume uninsulated wall properties if no exterior finish, I added some clarification.

Reply:
Thanks. Is this because we still model garage walls and gable end walls as wood-framed cavity walls even if primary exterior walls are masonry?

Reply:
Yes exactly, we have an issue open for it: https://github.com/NREL/resstock/issues/587